### DISCUSSION TITLE
Can't log in

### DISCUSSION THREAD

**Discussion by OmidRasouli** [cssbattle/forum#244] [Q&A]
Hi,
I can't log in to the site anymore. I tried everything, even I tried to log in with incognito or another browser but I couldn't.
I used Chrome and Firefox. It shows me "A password-less login link has been sent to your email..." but I don't get any email.

**Comment by chinchang** [comment]
Did you look for the login mail in Spam? otherwise please mail us your login email on contact@cssbattle.dev

**Reply by OmidRasouli** [reply]
Thanks for your reply.
Yes I did it and I tried Google, Github but they didn't work.
This error happens when I click on login with Google.
I will send you an email with the email of my account.

**Comment by chinchang** [comment]
Did you try checking the login link mail in spam?
Otherwise, please let us know your login email on ***@***.***

On Sun, 27 Mar 2022 at 3:57 PM, Omid ***@***.***> wrote:

> Hi,
> I can't log in to the site anymore. I tried everything, even I tried to
> log in with incognito or another browser but I couldn't.
> I used Chrome and Firefox. It shows me "A password-less login link has
> been sent to your email..." but I don't get any email.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/cssbattle/forum/discussions/244>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAC4YDTSITKMMJFJE4NCWETVCAZXLANCNFSM5RYSWIJA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
*Kushagra Gour*
kushagra.dev
@chinchang457 <https://twitter.com/chinchang457>

**Reply by OmidRasouli** [reply]
Thanks for your response.
I'm located in Iran and unfortunately, Iran is restricted in some of the services around the world, because of that I realized it's not possible to log in without a VPN (changing the IP).
